e2e/test/scenarios/filters-reproductions/dashboard-filters-reproductions.cy.spec.js

```diff
@@ -179,7 +178,7 @@ describe("issue 8030 + 32444", () => {
             cy.findByText(filterDetails.name).click();
             popover().within(() => {
               // the filter is connected only to the first card
-              multiAutocompleteInput().type("1{enter}");
+              cy.get("input").type("1{enter}");
```

Comment:
Reverts this
https://github.com/metabase/metabase/pull/43293/files#diff-9db584711720ed7f09df046944b2c4e4a71ba62532dccb03c89930cffaa9e779L180-R181